index.js

```diff
@@ -100,13 +108,34 @@ multiWorker.on('internalError', error =>
 multiWorker.on('multiWorkerAction', (verb, delay) =>
     winston.info(`*** checked for worker status: ${verb} (event loop delay: ${delay}ms)`));
 
+scheduler.on('start', () =>
+    winston.info('scheduler started'));
+scheduler.on('end', () =>
+    winston.info('scheduler ended'));
+scheduler.on('poll', () =>
+    winston.info('scheduler polling'));
+scheduler.on('master', state =>
+    winston.info(`scheduler became master ${state}`));
+scheduler.on('error', error =>
+    winston.info(`scheduler error >> ${error}`));
+scheduler.on('working_timestamp', timestamp =>
+    winston.info(`scheduler working timestamp ${timestamp}`));
+scheduler.on('transferred_job', (timestamp, job) =>
+    winston.info(`scheduler enquing job timestamp  >>  ${JSON.stringify(job)}`));
+
 multiWorker.start();
+scheduler.connect(() => {
+    scheduler.start();
```

Comment:
Don't know why the `start` syntax is different for `multiworker` and `scheduler`. But this is what they put in their doc.